index.js

```diff
@@ -56,10 +56,10 @@ var createServer = function (e, opts) {
 
     var toPlaylist = function () {
       var toEntry = function (file, i) {
-        return '#EXTINF:-1,' + file.path + '\n' + 'http://' + host + '/' + i
+        return '#EXTINF:-1,' + file.name + '\n' + 'http://' + host + '/' + file.name.replace(/\s/g, '')
```

Comment:
Can't open file if file.name is not ASCII.

Reply:
There is an alternative based on libtorrent written in python https://aur.archlinux.org/packages/bittorrent2player